client/my-sites/sharing/connections/account-dialog.jsx

```diff
@@ -1,186 +1,181 @@
+				conflicting={ account.isConnected && selectedAccount && this.areAccountsConflicting( account, selectedAccount ) }
+				onChange={ this.onSelectedAccountChanged.bind( null, account ) } />
+		);
+	}
 
-	getConnectedAccountsContent: function() {
-		var connectedAccounts = this.getAccountsByConnectedStatus( true );
+	getConnectedAccountsContent() {
+		const connectedAccounts = this.getAccountsByConnectedStatus( true );
 
 		if ( connectedAccounts.length ) {
 			return (
 				<div className="account-dialog__connected-accounts">
-					<h3 className="account-dialog__connected-accounts-heading">{ this.translate( 'Connected' ) }</h3>
+					<h3 className="account-dialog__connected-accounts-heading">{ this.props.translate( 'Connected' ) }</h3>
 					<ul className="account-dialog__accounts">
 						{ this.getAccountElements( connectedAccounts ) }
 					</ul>
 				</div>
 			);
 		}
-	},
+	}
 
-	getDisclaimerText: function() {
+	getDisclaimerText() {
 		if ( 1 === this.props.accounts.length ) {
 			// If a single account is available, show a simple confirmation
 			// prompt to ask the user to confirm their connection.
-			return this.translate( 'Confirm this is the account you would like to authorize. Note that your posts will be automatically shared to this account.', {
-				context: 'Sharing: Publicize connection confirmation'
-			} );
-		} else {
-			// Otherwise, we assume that multiple connections exist for a
-			// single Keyring connection, and the user must choose which
-			// account to connect.
-			return this.translate( 'Select the account you wish to authorize. Note that your posts will be shared to the selected account automatically.', {
+			return this.props.translate(
+				'Confirm this is the account you would like to authorize. Note that your posts will be automatically shared to this account.', {
+					context: 'Sharing: Publicize connection confirmation'
+				} );
+		}
+
+		// Otherwise, we assume that multiple connections exist for a
+		// single Keyring connection, and the user must choose which
+		// account to connect.
+		return this.props.translate(
+			'Select the account you wish to authorize. Note that your posts will be shared to the selected account automatically.', {
 				context: 'Sharing: Publicize connection confirmation'
 			} );
-		}
-	},
+	}
 
-	onClose: function( action ) {
-		var accountToConnect = this.getAccountToConnect();
+	onClose( action ) {
+		const accountToConnect = this.getAccountToConnect();
 
 		if ( 'connect' === action && accountToConnect ) {
 			this.props.onAccountSelected( this.props.service, accountToConnect.keyringConnectionId, accountToConnect.ID );
 		} else {
 			this.props.onAccountSelected();
 		}
-	},
+	}
 
-	render: function() {
-		var classes = classNames( 'account-dialog', {
-			'single-account': 1 === this.props.accounts.length
-		} ), buttons;
+	render() {
+		const classes = classNames( 'account-dialog', {
+				'single-account': 1 === this.props.accounts.length
+			} ),
+			buttons = [
```

Comment:
Might be more readable if this were a separate `const` declaration, not comma-separated, especially since both of these variables are multi-line. What do you think?